app/Resources/views/article-rds.html.twig

```diff
@@ -16,6 +20,14 @@
 
 {% endblock %}
 
+{% block meta %}
+
+    {{ parent() }}
+
+    <meta name="robots" content="noindex">
```

Comment:
I assume `index` vs. `noindex` and other possible values are out of scope for this PR